src/seeds.dao.cpp

```diff
@@ -950,17 +950,39 @@ void dao::send_mimic_delegatee_vote (const name & delegatee, const name & scope,
 
 }
 
+uint64_t dao::get_new_moon (uint64_t timestamp) {
+
+  moon_phases_tables moonphases_t(contracts::scheduler, contracts::scheduler.value);
+
+  auto mitr = moonphases_t.lower_bound(timestamp);
+
+  while (mitr != moonphases_t.end()) {
+    if (mitr->phase_name == "New Moon") {
+      return mitr->timestamp;
+    }
+    mitr++;
+  }
+
+  return 0;
+
+}
+
 void dao::init_cycle_new_stats () {
 
   cycle_tables cycle_t(get_self(), get_self().value);
   cycle_table c_t = cycle_t.get();
 
   cycle_stats_tables cyclestats_t(get_self(), get_self().value);
 
+  uint64_t now = eosio::current_time_point().sec_since_epoch() - (utils::seconds_per_day * 5);
+
+  uint64_t start_time = get_new_moon(now);
```

Comment:
make it like this:

uint64_t start_time  = get_new_moon(start_time - utils::seconds_per_day);

This way we can be sure we get the current new moon... it may be a few seconds off...